app/styles/base/_typography.scss

```diff
@@ -516,6 +516,9 @@ tr {
 // --------------------------------------------------
 .draw-tools--container {
   position: relative;
+  width: 150px;
+  height: 33.21px;
+  left: 25px;
```

Comment:
Next step is to start the mobile compatibility. Not MVP but let's keep those problems in mind. Pixel dimensions are red flags